OWNERS

```diff
@@ -1,6 +1,8 @@
 reviewers:
   - codificat
   - mazzystr
+  - jean-edouard
```

Comment:
Hi Jed, I've been running KubeVirt for awhile now and haven't heard of you.  I reached out to several senior members to see if they vouch for you.  

In the meantime can you create a page under `kubevirt/community/member_dd` and fill out the member due diligence checklist ([link](https://github.com/kubevirt/community/blob/master/membership_checklist.md))?

Reply:
Hi Chris! I'm not sure which proposals you are referring to, but I'm applying to be a reviewer to ultimately become approver.
I thought it would be a good idea after 3 kubevirt/kubevirt approvers, including myself, tried and failed to approve https://github.com/kubevirt/community/pull/119
Please let me know if you need anything else!

Reply:
Sounds good.  Let's try it out and see how it goes